frontend/src/components/yellowbutton/YellowButton.module.css

```diff
@@ -0,0 +1,23 @@
+@import url('https://fonts.googleapis.com/css2?family=Hammersmith+One&family=Roboto+Mono:ital,wght@0,100..700;1,100..700&display=swap');
+button {
+    background: #FEC126;
+    border-radius: 0.3rem;
```

Comment:
i recommend using px for radii so that the button doesnt appear more or less round on different devices